src/views/privacypolicy-apps/privacypolicy-apps.jsx

```diff
@@ -0,0 +1,313 @@
+const React = require('react');
+
+const FormattedMessage = require('react-intl').FormattedMessage;
+const Page = require('../../components/page/www/page.jsx');
+const render = require('../../lib/render.jsx');
+const injectIntl = require('react-intl').injectIntl;
+const intlShape = require('react-intl').intlShape;
```

Comment:
In order to avoid saying `require('react-intl')` multiple times I suggest something like this:
```suggestion
const {FormattedMessage, injectIntl, intlShape} = require('react-intl');
```